resources/Microsoft.Windows.Setting.Time/Microsoft.Windows.Setting.Time.psd1

```diff
@@ -0,0 +1,131 @@
+    RootModule           = 'Microsoft.Windows.Setting.Time.psm1'
+
+    # Version number of this module.
+    ModuleVersion        = '0.1.0'
+
+    # Supported PSEditions
+    # CompatiblePSEditions = @()
+
+    # ID used to uniquely identify this module
+    GUID                 = '6a947f86-eb17-46a4-9b0d-9f757b19c29a'
+
+    # Author of this module
+    Author               = 'Microsoft Corporation'
+
+    # Company or vendor of this module
+    CompanyName          = 'Microsoft Corporation'
+
+    # Copyright statement for this module
+    Copyright            = '(c) Microsoft Corporation. All rights reserved.'
+
+    # Description of the functionality provided by this module
+    Description          = 'DSC Resource for Windows Time Settings'
+
+    # Minimum version of the PowerShell engine required by this module
+    PowerShellVersion    = '7.2'
+
+    # Name of the PowerShell host required by this module
+    # PowerShellHostName = ''
+
+    # Minimum version of the PowerShell host required by this module
+    # PowerShellHostVersion = ''
+
+    # Minimum version of Microsoft .NET Framework required by this module. This prerequisite is valid for the PowerShell Desktop edition only.
+    # DotNetFrameworkVersion = ''
+
+    # Minimum version of the common language runtime (CLR) required by this module. This prerequisite is valid for the PowerShell Desktop edition only.
+    # ClrVersion = ''
+
+    # Processor architecture (None, X86, Amd64) required by this module
+    # ProcessorArchitecture = ''
+
+    # Modules that must be imported into the global environment prior to importing this module
+    # RequiredModules = @()
+
+    # Assemblies that must be loaded prior to importing this module
+    # RequiredAssemblies = @()
+
+    # Script files (.ps1) that are run in the caller's environment prior to importing this module.
+    # ScriptsToProcess = @()
+
+    # Type files (.ps1xml) to be loaded when importing this module
+    # TypesToProcess = @()
+
+    # Format files (.ps1xml) to be loaded when importing this module
+    # FormatsToProcess = @()
+
+    # Modules to import as nested modules of the module specified in RootModule/ModuleToProcess
+    # NestedModules = @()
+
+    # Functions to export from this module, for best performance, do not use wildcards and do not delete the entry, use an empty array if there are no functions to export.
+    FunctionsToExport    = '*'
+
+    # Cmdlets to export from this module, for best performance, do not use wildcards and do not delete the entry, use an empty array if there are no cmdlets to export.
+    CmdletsToExport      = '*'
+
+    # Variables to export from this module
+    VariablesToExport    = '*'
+
+    # Aliases to export from this module, for best performance, do not use wildcards and do not delete the entry, use an empty array if there are no aliases to export.
+    AliasesToExport      = '*'
+
+    # DSC resources to export from this module
+    DscResourcesToExport = @('Time')
```

Comment:
If you're not going to add it in this PR, once this PR merges I would request that you add a follow up issue to track the addition of adding clocks for different timezones under `HKEY_CURRENT_USER\Control Panel\TimeDate\AdditionalClocks\1`

![image](https://github.com/user-attachments/assets/f5676ae5-d8b1-4509-a7b4-c93446668351)

Reply:
I'll put it on #109

Reply:
@Trenly I have partially started on this one. Do you mind sharing your thoughts on #100 how we are going to set both 1 and 2 in the registry?

Reply:
I would implement it as a single `AdditionalClock` resource, which has Properties of `Ensure`, `TimeZone`, `DisplayName`. 

For `Get` -
* Fetch the values in `1`
* Compare against properties
* If the properties match, set ensure present and return the current state
* If the properties don't match, fetch the values in `2`
* Compare against properties
* If the properties match, set ensure present; otherwise, set ensure absent
* Return the current state

For `Test` - 
* Run `Get`
* Compare `Ensure` of the current state to `Ensure` of the desired state
* Return result of comparison

For `Set` it's a bit more complicated, I'd imagine you'd first want to determine if either `1` or `2` had a matching `DisplayName`, if it does, set the values for that clock. If no display names match, check if either timezone matches, if it does, set the values for that clock. If neither are a match, put it in slot 1 if the enable property of the slot is not set. If slot 1 is already enabled, put it in slot 2 if the enable property of the slot is not set. If slot 2 is already enabled, throw an error that the maximum number of additional clocks have been set.

-----

Alternatively, you could just add a parameter for `Id` and restrict it to being `1` or `2` and have it be mandatory for `Set` operations but optional for `Get` and `Test`.  That's honestly probably the better option

Reply:
Thanks for the details. One small question, how will users be able to specify both 1 and 2 in a single configuration document?

Reply:
They call the resource twice - once for setting slot one, and a second time for setting slot two

Reply:
I lack the knowledge on this part. Either clock 1 or clock 2 can be turned on, or both. Shouldn't the user be able to say something like: `AdditionalClock1Settings` and `AdditionaClock2Settings`?

Reply:
No need; They can confugure like:
```Yaml
    - resource: Microsoft.Windows.Setting.Time/AdditionalClock
      directives:
        description: Set clock 1
        allowPrerelease: true
      settings:
        Slot: 1
        DisplayName: UTC
        Timezone: UTC
        Ensure: Present
    - resource: Microsoft.Windows.Setting.Time/AdditionalClock
      directives:
        description: Set clock 2
        allowPrerelease: true
      settings:
        Slot: 2
        DisplayName: Current time in Redmond
        Timezone: Pacific Standard Time
        Ensure: Present
```

Reply:
Ah cool. I'll pick it up after this is merged!